packages/mobile/locales/en-US/paymentRequestFlow.json

```diff
@@ -1,10 +1,13 @@
 {
   "requests": "Requests",
   "empty": "You have no Payment Requests",
-  "paymentRequests": "Payment Requests",
+  "incomingPaymentRequests": "Requests Waiting on You",
+  "outgoingPaymentRequests": "Payments You’ve Requested",
   "celoDollarBalance": "{{CeloDollar}} Balance",
   "pay": "Pay",
   "decline": "Decline",
+  "remind": "Remind",
+  "cancel": "Cancel",
```

Comment:
Nit: for one word translations that will likely be used elsewhere, please put them in the `global.json` file. You'll find there's already a cancel translation there.